e2echart/e2e-chart-template.html

```diff
@@ -464,6 +494,12 @@ <h5 class="modal-title">Resource</h5>
         timelineGroups.push({group: "endpoint-availability", data: []})
         createTimelineData(disruptionValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isEndpointConnectivity, regex)
 
+        timelineGroups.push({group: "apiserver-disruption", data: []})
```

Comment:
Will ask on the call today but I'm curious why we want to fork to a new section, I'd love to mix the new data into the current disruption monitor framework just with adjusting the names of the backends.

Reply:
yes, i removed `apiserver-disruption`, all disruptions will show up in `endpoint-availability`